src/store/zoom/index.ts

```diff
@@ -15,30 +15,16 @@ function handleCalculateZoom(
   step,
   disablePadding,
   getTarget,
-  isBtnFunction,
 ) {
   const {
     scale,
     options: { maxScale, minScale },
     scalePadding: { size, disabled },
-    wrapperComponent,
   } = this.stateProvider;
 
-  let targetScale = null;
-
-  if (isBtnFunction) {
-    const scaleFactor = window.innerWidth * 0.0001;
-    const zoomFactor = delta < 0 ? 30 : 20;
-    targetScale =
-      scale + (step - step * scaleFactor) * delta * (scale / zoomFactor);
-  } else {
-    const wrapperToWindowScale =
-      2 - window.innerWidth / wrapperComponent.offsetWidth;
```

Comment:
@prc5 could you possibly comment on the reason why `window.innerWidth` is involved in the scale calculation? Is that needed for certain devices or use cases? Understanding what this does could help us decide how to best work with this, or work around it.

I also see issue #141. In my app, my zoomable element is sometimes full screen width and sometimes only partial width. I configured the zoom speed for when it is full width. But then when it's partial width, the zoom rate becomes huge and uncontrollable. This line seems to be the part of the calculation that changes drastically for those cases. Could come out to `2 - 1 = 1` when the wrapper is full window width, or something like `2 - 3 = -1` if the wrapper were for example 1/3 of the window width. So then `scaleFactor` changes from 0.99 to 0.2. Seems unintentional?

For context, the code appears to have been introduced here:
https://github.com/prc5/react-zoom-pan-pinch/commit/f01003f7e26a55c03e53c6e06134213a1daba402#diff-d36944379ebb91642a3a88b44a14aa524dcbca38bccd52fd968872cfe396e0a8R186